lib/msal-node/docs/certificate-credentials.md

```diff
@@ -191,36 +141,48 @@ const forge = require('node-forge');
  */
 function convertPFX(pfx, passphrase = null) {
 
-    const asn = forge.asn1.fromDer(forge.util.decode64(pfx));   
+    const asn = forge.asn1.fromDer(forge.util.decode64(pfx));
     const p12 = forge.pkcs12.pkcs12FromAsn1(asn, true, passphrase);
-    
+
     // Retrieve key data
     const keyData = p12.getBags({ bagType: forge.pki.oids.pkcs8ShroudedKeyBag })[forge.pki.oids.pkcs8ShroudedKeyBag]
         .concat(p12.getBags({ bagType: forge.pki.oids.keyBag })[forge.pki.oids.keyBag]);
-    
+
     // Retrieve certificate data
     const certBags = p12.getBags({ bagType: forge.pki.oids.certBag })[forge.pki.oids.certBag];
     const certificate = forge.pki.certificateToPem(certBags[0].cert)
-    
+
     // Convert a Forge private key to an ASN.1 RSAPrivateKey
     const rsaPrivateKey = forge.pki.privateKeyToAsn1(keyData[0].key);
-    
+
     // Wrap an RSAPrivateKey ASN.1 object in a PKCS#8 ASN.1 PrivateKeyInfo
     const privateKeyInfo = forge.pki.wrapRsaPrivateKey(rsaPrivateKey);
-    
+
     // Convert a PKCS#8 ASN.1 PrivateKeyInfo to PEM
     const privateKey = forge.pki.privateKeyInfoToPem(privateKeyInfo);
-    
+
     console.log("Converted certificate: \n", certificate);
     console.log("Converted key: \n", privateKey);
-    
+
     return {
         certificate: certificate,
         key: privateKey
     };
 }
 ```
 
+### (Optional) Creating an HTTPS server
+
+The OAuth 2.0 protocol recommends using an HTTPS connection whenever possible. Most cloud services like Azure App Service will provide HTTPS connection by default via proxy. If for testing purposes you would like to setup your own HTTPS server, see the [Node.js HTTPS guide](https://nodejs.org/knowledge/HTTP/servers/how-to-create-a-HTTPS-server/).
+
+You'll also need to add your self-signed certificates to the *credential manager* / *key chain* of your **OS** to bypass the browser's security policy. You may still see a warning in your browser afterwards (e.g. Chrome).
+
+* For Windows users, follow the guide here: [How to: View certificates with the MMC snap-in](https://docs.microsoft.com/dotnet/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in).
+
+* For Linux and MacOS users, use community guides -with judgment- to learn about how to install certificates.
```

Comment:
Consider: "For Linux and MacOS users, please consult your operating system documentation on how to install certificates."